src/ray/common/grpc_util.h

```diff
@@ -28,6 +28,14 @@
 
 namespace ray {
 
+template <typename, typename = void>
```

Comment:
Let's put it under `type_traits.h`
https://github.com/ray-project/ray/blob/master/src/ray/util/type_traits.h